modular_skyrat/modules/mapping/code/areas/away_content.dm

```diff
@@ -162,61 +162,65 @@
 	name = "Black Mesa HECU Territory External Hall"
 	icon_state = "awaycontent10"
 
+/area/awaymission/black_mesa/hecu_zone_infirmary
+	name = "Black Mesa HECU Territory Infirmary"
+	icon_state = "awaycontent11"
+
 /area/awaymission/black_mesa/hecu_zone_bathroom
 	name = "Black Mesa HECU Territory Bathroom"
-	icon_state = "awaycontent11"
+	icon_state = "awaycontent12"
```

Comment:
These icon states, I do not like them, no I do not

No clue if they're worth fixing though as I've never seen them used outside this mission